pubnub-chat-impl/src/commonTest/kotlin/com/pubnub/kmp/ChatTest.kt

```diff
@@ -1655,4 +1702,19 @@ class ChatTest : BaseTest() {
         verify { pubnub.destroy() }
         verify { timerManager.destroy() }
     }
+
+    @Test
+    fun disconnectSubscriptions_withNoActiveSubscriptions_shouldStillWork() {
```

Comment:
I see there's a `disconnectSubscriptions_shouldCallPubNubDisconnect` test in ChatTest.kt - what's the difference between them?